src/taxonomy/api/hooks/selectors.js

```diff
@@ -18,3 +21,7 @@ export const useTaxonomyListDataResponse = () => {
 export const useIsTaxonomyListDataLoaded = () => (
   useTaxonomyListData().status === 'success'
 );
+
+export const callExportTaxonomy = (pk, format) => (
+  exportTaxonomy(pk, format)
+);
```

Comment:
I don't think selectors is the correct place for this, although I think this might be a preexisting problem with this module. 

I think it's important to keep this ADR in mind: https://github.com/openedx/frontend-template-application/blob/master/docs/decisions/0002-feature-based-application-organization.rst

None of the functions here are hooks or selectors in the commonly understood context of those terms.

Reply:
@xitij2000 I updated the UI components to match with the ADR here: b09b86a2caf2ab3d26f56097725b02d48de80120

About the data layer, we have try to use [useQuery](https://tanstack.com/query/v4/docs/react/reference/useQuery) to get data. On df8432dbe9c86b5bebb35fc5e158d20b98024bd6 I tried to match with the structure on the ADR, @xitij2000 could you check it?

CC @bradenmacdonald

Reply:
Given that this is using useQuery instead of redux I'm not sure how things would map. However, I think it's best to avoid confusion with exist terms like hooks, thunks, selectors, if you are not using those since someone else working on this might have difficulty navigating this codebase in that case.

Reply:
Thanks! I understand. I deleted `thunks.js` and I move `hooks.jsx` to `api/apiHooks.jsx` (to avoid use hooks or any other exist term). Also [I added a docstring](https://github.com/openedx/frontend-app-course-authoring/commit/080b03ec36a5d019f475bf31a9a5fe5791973e9b#diff-b5aca46925ac5f8adac9fbe39002822044019f623908e73f3dba1d5386048dcdR3) in `api/apiHooks.jsx` to explain all of this.